.github/workflows/reusable-release.yml

```diff
@@ -186,6 +186,83 @@ jobs:
           git merge release-candidate
           git push
 
+  publish_rc_followup:
```

Comment:
this is a total copy of the `rc` workflow, the only difference is the one line where we run the NPM script to publish the release.  Maybe (hopefully) there's a way to de-dupe this code?

Reply:
Thanks for pointing out the difference in the flows. I think we might be able to reuse the existing `rc` flow for both options.

Maybe something like this on line 114:
```
    if: ${{ inputs.release_type == 'rc' || inputs.release_type == 'rc-followup' }}
```
And something like this on line 168:
```
RELEASE_TYPE=${{ inputs.release_type }}
if [[ $RELEASE_TYPE == "rc" ]]; then
    npm run publish:release-candidate -- --yes
else
    npm run publish:release-candidate-followup -- --yes
fi
```